packages/paste-website/src/components/tokens-list/types.ts

```diff
@@ -1,26 +1,79 @@
-export interface Token {
-  name: string;
-  value: string;
+import type {
```

Comment:
ℹ️ Updated types to match the new tokens data format (since we are now using the build from our tokens package). More on this in the PR description.